README.md

```diff
@@ -118,3 +118,11 @@ If someone has made a pull request and you would like to add code to their branc
   .
   git push <your-fork> <branch-name>
   ```
+  
+  ## Development best practises
+  
+  ### Testing
+  
+  We do have both functional level testing ([nightwatch](http://nightwatchjs.org/)), but also more component/unit testing ((jest)[https://jestjs.io/]).
```

Comment:
Improving the English:

We have functional testing with [nightwatch](http://nightwatchjs.org/), and component/unit testing with [jest](https://jestjs.io/).

Reply:
👍